math/mathcore/test/testTMath.cxx

```diff
@@ -141,6 +141,20 @@ void testPlane()
         << endl;
 }
 
+void testBreitWignerRelativistic(){
```

Comment:
The test is just printing the values. It would be nice to have a new executable and not included in testTMath where gtest is used. 
In this case we compare obtained values with some reference ones which can be hard-coded in the test. 
Here is an example: https://github.com/root-project/root/blob/master/math/mathcore/test/testKahan.cxx